client/js/helpers/contextMenu.js

```diff
@@ -170,10 +170,12 @@ export function generateChannelContextMenu($root, channel, network) {
 }
 
 export function generateUserContextMenu($root, channel, network, user) {
-	const currentChannelUser = channel.users.find((u) => u.nick === network.nick) || {};
+	const currentChannelUser = channel
```

Comment:
`(channel && channel.users.find((u) => u.nick === network.nick)) || {}` is slightly cleaner